CMU-313/nodebb-f24-slackers
# Merge project 1 changes

# Description
Refactored code depth of line 71-74 of src/user/uploads.js to reduce code depth from 4 to 3. This was done by abstracting the inner arrow function into a seperate function.

Soundcloud issue [here.](https://sonarcloud.io/project/issues?cleanCodeAttributeCategories=ADAPTABLE&issueStatuses=OPEN%2CCONFIRMED&id=CMU-313_NodeBB&open=AZFmjAJDybYwxy-_uEcF)

# UI Manual Test
1. Create a topic
2. Upload an image to the topic
3. Navigate to user tab
4. Click uploads and see that there is one upload for your profile
5. Click delete beside the upload and note the console log in ./nodebb log
<img width="1511" alt="Screenshot 2024-08-28 at 7 16 06 PM" src="https://github.com/user-attachments/assets/289964be-9dd3-4bdb-88ba-d8cc627c630f">

# Coverage Test
This is covered by coverage tests as shown:
<img width="966" alt="Screenshot 2024-08-28 at 6 03 01 PM" src="https://github.com/user-attachments/assets/b06a8445-faf0-4bbb-972e-caa2434ace90">
